Tools/CISupport/ews-build/steps.py

```diff
@@ -3585,6 +3585,7 @@ class RunJavaScriptCoreTests(shell.Test, AddToLogMixin):
 
     def __init__(self, **kwargs):
         super().__init__(logEnviron=False, sigtermTime=10, **kwargs)
+        kwargs['timeout'] = 2 * 60 * 60
```

Comment:
This timeout doesn't work, since kwargs is not used after setting it here. It should be done before the super() call. 
However, let's just maintain consistency with rest of this file and pass timeout as a function parameter to super().__init__().